src/resource_format/kt_resource_format_saver.cpp

```diff
@@ -16,7 +15,17 @@ bool KtResourceFormatSaver::recognize(const Ref<Resource>& p_resource) const {
 Error KtResourceFormatSaver::save(const Ref<Resource>& p_resource, const String& p_path, uint32_t p_flags) {
     Ref<KotlinScript> script = p_resource;
     ERR_FAIL_COND_V(script.is_null(), ERR_INVALID_PARAMETER);
-    String source = script->get_source_code();
+
+    String package {p_path.replace("src/main/kotlin/", "")
```

Comment:
I suppose this will not work for locations outside of `src/main/kotlin` like a custom source set?

Reply:
I suppose it won't. I didn't really pay attention to that part, that code was simply moved from somewhere else. 
As it is now, it doesn't have any effect as there is no "Kotlin source code" right now. We are using .gdj, not .kt.
I left it here, because it's going to be relevant for the next PR supposed to fix the script creation UI.